spec/features/idv/analytics_spec.rb

```diff
@@ -4,6 +4,8 @@
   include IdvStepHelper
   include InPersonHelper
 
+  same_address = rand(2) == 1
```

Comment:
Yeah, I definitely was thinking this is _probably_ a bad idea, glad to know I wasn't off base in thinking that. We do have a separate test suite for the `address_step` code that does confirm that the value can be set to true or false. I think it's reasonable to assume that since the `address_step` test passes, and a "true" value makes it's way through the analytics that the false value would do the same. Let me know if my reasoning is off base or if I'm missing something @zachmargolis

Reply:
Yup just a "true" makes sense

Reply:
I noticed that this randomness is combined with a new conditional in one of the test steps. I would generally caution against using conditionals in tests wherever possible because they expose the test to a higher likelihood of bugs relative to the code that they test against.

Any exception to that should be thought out very carefully and lack a reasonable non-conditional alternative.